src/arc.ts

```diff
@@ -35,7 +35,8 @@ export function targetedNetwork(): Networks {
       return "rinkeby";
     }
     case "mainnet":
-    case "production" : {
+    case "production":
+    case undefined : {
```

Comment:
i think the fix is case 'main' actually, look at package.json... unless you are sure that it is undefined